.github/workflows/ci.yml

```diff
@@ -19,6 +19,33 @@ jobs:
       - name: Run tests
         run: bundle exec rake test:units
 
+  test-legacy:
+    runs-on: ubuntu-20.04
```

Comment:
🗒️ Use the older Ubuntu runner that is compatible with Ruby `"2.0", "2.1", "2.2"`